src/Main/AlexanderOnTest.NetCoreWebDriverFactory/IWebDriverFactory.cs

```diff
@@ -45,8 +46,9 @@ Uri GridUri
         /// <param name="platformType"></param>
         /// <param name="windowSize"></param>
         /// <param name="headless"></param>
+        /// <param name="windowCustomSize"></param>
         /// <returns></returns>
-        IWebDriver GetRemoteWebDriver(Browser browser, PlatformType platformType = PlatformType.Any, WindowSize windowSize = WindowSize.Hd, bool headless = false);
+        IWebDriver GetRemoteWebDriver(Browser browser, PlatformType platformType = PlatformType.Any, WindowSize windowSize = WindowSize.Hd, bool headless = false, Size windowCustomSize = new Size());
```

Comment:
Given that I am already changing these signatures, I think I prefer to change to using WindowSize.Custom as default.
`IWebDriver GetRemoteWebDriver(Browser browser, PlatformType platformType = PlatformType.Any, WindowSize windowSize = WindowSize.Custom, bool headless = false, Size windowCustomSize = new Size(1366, 768));`

Reply:
Doh! of course that is not a compile time constant. Will have to stay as is to keep v2.x functionality